src/rustllvm/RustWrapper.cpp

```diff
@@ -985,24 +985,35 @@ extern "C" int64_t LLVMRustDIBuilderCreateOpPlusUconst() {
   return dwarf::DW_OP_plus_uconst;
 }
 
+// Adds YK labels as close to the beginning of BasicBlocks as possible.
+//
+// This is tricky because LLVM has certain constraints which the bitcode
+// checker will complain about. For example, you may not insert debug info
+// before PHI nodes.
```

Comment:
Are there any others?

Reply:
@ptersilie do you remember?

Reply:
I don't remember if there were others, though it wouldn't surprise me if there are.